cips/cip-1.md

```diff
@@ -311,9 +312,8 @@ format. There is a [CIP template](./cip-template.md) to follow.
 
 ## CIP Header Preamble
 
-Each CIP must begin with an RFC 822 style header preamble, preceded
-and followed by three hyphens (---). This header is also termed
-“front matter” by Jekyll. The headers must appear in the following order.
+Each CIP must begin with an RFC 822 style header preamble.
```

Comment:
why remove the rule on hyphens? this is (---) how we construct the preamble. Observe the CIP template doc for example.

Reply:
I won't approve this section.

Reply:
Ok, I didn't convince you, then I reverted this section and changed the order of the preamble header instead.